Source/WTF/wtf/NativePromise.h

```diff
@@ -1385,6 +1409,12 @@ class NativePromiseProducer final : public ConvertibleToNativePromise {
         m_promise->template chainTo<ResolveValueT2, RejectValueT2, options2>(WTFMove(chainedPromise), callSite);
     }
 
+    template<typename RejectValueType_, typename = std::enable_if<AutoRejectNonVoid>>
+    void setDefaultReject(RejectValueType_&& rejectValue)
```

Comment:
Why do we need this setter?

Reply:
I felt a good things to have if along the lifecycle of the producer we ever wanted to change the reject value